modules/email/src/test/java/com/intuit/wasabi/email/EmailModuleTest.java

```diff
@@ -20,9 +20,7 @@
 import com.google.inject.Injector;
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
concern re the wildcard import?

this is default intellij ... which again i seldom-if-ever do anything w the imports directly.

defer for broader discussion